siobrultech_protocols/gem/protocol.py

```diff
@@ -49,25 +79,54 @@ def __init__(
     def connection_made(self, transport: asyncio.BaseTransport) -> None:
         LOG.info("%d: Connection opened", id(self))
         self._transport = transport
```

Comment:
Should we add a `assert` now that `self._transport` is currently `None` before setting it here?

Reply:
Yep